framework/build.gradle

```diff
@@ -7,9 +7,21 @@ ext {
 }
 sourceSets {
     main {
+        java {
+            // NO-AFU
+            exclude '**/org/checkerframework/common/wholeprograminference/**'
+            exclude '**/org/checkerframework/framework/stub/AddAnnotatedFor.java'
+            exclude '**/org/checkerframework/framework/stub/ToIndexFileConverter.java'
+        }
+
         resources {
             // Stub files, message.properties, etc.
             srcDirs += ['src/main/java', "${buildDir}/generated/resources"]
+
+            // NO-AFU
+            exclude '**/org/checkerframework/common/wholeprograminference/**'
+            exclude '**/org/checkerframework/framework/stub/AddAnnotatedFor.java'
```

Comment:
Same comment as before about excluding Java files from the resources.

Reply:
I removed these lines in `checker`, but when I do the same here, the javadoc build fails. I'll investigate separately.